### PR TITLE
Quiz Functionality

### PR DESCRIPTION
**Pull request by nikhiljugale007**
Added quiz result page.
Added mock backend of categories.
Added more quizzes in backend

### CODE REVIEW

File: src/pages/quiz-solution-page/QuizSolution.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	justify-self: center;
+	font-size: 2rem;
+	padding: 2rem;
+	border: 2px solid #000;
+	border-radius: 10px;
+}
+.two-column {
+	display: grid;
+	grid-template-columns: 1fr 10px 1fr;
+	/* background-color: red; */
```
Comment (yaswanthmyneni) [inline src/pages/quiz-solution-page/QuizSolution.css]: changes: remove commented code

**Review by yaswanthmyneni — APPROVED** [review]
The code is written well, you can merge after changes.

File: src/pages/quiz-solution-page/QuizSolution.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	display: grid;
+	gap: 1.6rem;
+	grid-template-rows: auto 1fr auto;
+}
+.score-container {
+	width: 35rem;
+	justify-self: center;
+	font-size: 2rem;
+	padding: 2rem;
+	border: 2px solid #000;
```
Comment (yaswanthmyneni) [inline src/pages/quiz-solution-page/QuizSolution.css]: changes: use css variables for colors

File: src/pages/quiz-solution-page/QuizSolution.css (hunk 1 above)
Comment (nikhiljugale007) [reply src/pages/quiz-solution-page/QuizSolution.css]: removed

File: src/pages/quiz-solution-page/QuizSolution.css (hunk 2 above)
Comment (nikhiljugale007) [reply src/pages/quiz-solution-page/QuizSolution.css]: done

**Review by Ankur9669 — COMMENTED** [review]
Looks good to merge

**Pull request merged**